api/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:12.19-alpine3.10
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
```

Comment:
We probably want to remove this comment since we aren't using a wildcard.

Reply:
That's true. I'll remove that !

Reply:
If we remove this old comment about using wildcards we can then get this merged.

Reply:
@DanielAJoyce what's the reason we aren't using a wildcard here? Don't we want to copy the lock file too?